Tests/MapboxDirectionsTests/RouteRefreshTests.swift

```diff
@@ -113,11 +113,11 @@ class RouteRefreshTests: XCTestCase {
                 }
                 
                 let route = routeResponse.routes?[routeIndex]
-                route?.refreshLegAttributes(from: refresh.route)
-                route?.refreshLegIncidents(from: refresh.route)
+                route?.refresh(from: refresh.route)
```

Comment:
Should we also add separate tests for `refreshLegClosures` only if there is separate public method `Route.refreshLegClosures(from:legIndex:legShapeIndex:)`?

Reply:
I don't think it will bring some value, since `refresh(from:)` is basically a convenience wrapper method